### ISSUE TITLE
Circle description always visble

### ISSUE THREAD

**Issue by ChristophWurst** [nextcloud/contacts#3599]
### Steps to reproduce

1. Open the app
2. Go to the left sidebar/nav
3. Scroll down

### Expected behavior

Clean UI

### Actual behavior

![Bildschirmfoto vom 2023-09-05 14-01-44](https://github.com/nextcloud/contacts/assets/1374172/9b63fbac-6023-4243-99d1-855bdf61b9e2)

### Web server

None

### Database engine version

None

### PHP engine version

None

### Updated from an older installed version or fresh install

None

**Comment by ChristophWurst** [comment]
Text changed with https://github.com/nextcloud/contacts/pull/3437 but seemed to have always been rendered there.

Should this be inside an *info* icon/popover?

cc @ArtificialOwl @jancborchardt @marcoambrosini @nimishavijay

**Comment by jancborchardt** [comment]
Right, with the new much longer text this is a bit strange indeed.

> Should this be inside an info icon/popover?

@ChristophWurst sounds good, can be next to the "+" icon.

**Comment by call-me-matt** [comment]
maybe keep this text when no circles are created and (re)move it as soon as its being used?
